web/packages/teleterm/src/ui/DocumentAccessRequests/ReviewAccessRequest/useReviewAccessRequest.ts

```diff
@@ -70,9 +70,12 @@ export function useReviewAccessRequest({
     )
   );
   const [deleteRequestAttempt, runDeleteRequest] = useAsync(() =>
-    retry(() =>
-      ctx.clustersService.deleteAccessRequest(rootClusterUri, requestId)
-    )
+    retry(async () => {
+      await ctx.tshd.deleteAccessRequest({
+        rootClusterUri,
+        accessRequestId: requestId,
+      });
+    })
```

Comment:
No, we didn't not forget.

`protobuf-ts` [type exposes only](https://github.com/timostamm/protobuf-ts/blob/1798e0d43c31eafb4b5877ef72f05abb87456823/packages/runtime-rpc/src/unary-call.ts#L93) `.then` - this concept is called thenable https://masteringjs.io/tutorials/fundamentals/thenable.

To simplify this call we could remove `await`:
```ts
  const [deleteRequestAttempt, runDeleteRequest] = useAsync(() =>
    retry(async () =>
      ctx.tshd.deleteAccessRequest({
        rootClusterUri,
        accessRequestId: requestId,
      })
    )
  );
```
but I wanted the attempt to be of type `Attempt<void>`, since the we are not interested in the response anyway.